src/server/CookingApp/Controllers/RecipeController.cs

```diff
@@ -12,7 +12,8 @@ public class RecipeController(IRecipeService recipeService, IHttpContextAccessor
         [HttpPost("create-recipe")]
         public async Task<IActionResult> CreateRecipe([FromBody] string request)
         {
-            var recipeId = await recipeService.CreateRecipe(request, GetUser.ProfileId(httpContext));
```

Comment:
Why?

Reply:
I have the habit to do this in my code because my zoom is bigger. Fixing it right now.